config/install/group.role.idea-outsider.yml

```diff
@@ -10,5 +10,4 @@ internal: true
 audience: outsider
 group_type: idea
 permissions_ui: true
-permissions:
-  - 'view group_node:idea entity'
+permissions: {  }
```

Comment:
It should not be a part of the default configuration.
This change should be applied only when "OpenideaL Flavours" module is enabled.